Contents/Code/__init__.py

```diff
@@ -126,140 +216,265 @@ def Update(self, metadata, media, lang, force, movie):
         flags = flags | Prefs['hideSpoilerTags']    << 4 #0b10000 : Hide Plot Spoiler Tags
 
         if movie:
-            series = HttpReq("api/serie/fromep?id=%s&level=3&allpics=1&tagfilter=%d" % (aid, flags))
-            movie_episode_data = HttpReq("api/ep?id=%s" % (aid))
-            aid = try_get(series, 'id', None)
-            if not aid:
-                Log('Error! Series not found.')
-                return
-
-            if movie_episode_data['name'] == 'Complete Movie':
-                movie_name = series['name']
-                movie_sort_name = series['name']
-                movie_rating = float(series['rating'])
+            # Get series data
+            series_data = {}
+            series_data['shoko'] = HttpReq('api/v3/Episode/%s/Series' % aid) # http://127.0.0.1:8111/api/v3/Series/24
+            series_id = try_get(series_data['shoko'], 'IDs')
+            series_id = try_get(series_id, 'ID')
+            series_data['anidb'] = HttpReq('api/v3/Series/%s/AniDB' % series_id) # http://127.0.0.1:8111/api/v3/Series/24/AniDB
```

Comment:
Why create two indexes in an array, instead of just use two independent variables?|

```suggestion
            shoko_info = HttpReq('api/v3/Episode/%s/Series' % aid) # http://127.0.0.1:8111/api/v3/Series/24
            series_id = try_get(series_data['shoko'], 'IDs')
            series_id = try_get(series_id, 'ID')
            anidb_info = HttpReq('api/v3/Series/%s/AniDB' % series_id) # http://127.0.0.1:8111/api/v3/Series/24/AniDB
```

Reply:
Change the API call to get both data at once. In all places